db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+# Model for Race
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
Consider adding `null=True` to the `description` field in the `Race` model if you want to allow storing NULL in the database. Currently, it only allows empty strings due to `blank=True`.